tastyworks/example.py

```diff
@@ -1,73 +1,212 @@
+        # LOGGER.info('> '
+        #             f'{option.expiration_date}\t'
+        #             f'{option.strike}\t'
+        #             f'{option.option_type.value}\t'
+        #             f'>>\tTicker {option.get_dxfeed_symbol()}')
+        # options_tickers.append(option.get_dxfeed_symbol())
+        option.get_dxfeed_symbol()
+        options.append(option)
+
+    # Get the data for all created symbols via the streamer by subscribing
+    options_symbols = [options[x].symbol for x in range(options.__len__())]
+    streamer_list = {
+        # "Quote": [".SPY210419P410"]
+        "Quote": options_symbols
+    }
+    await streamer.add_data_sub(streamer_list)
+
+    # fetch the current data one time via streamer
+    feed_data = []
+    async for item in streamer.listen():
+        # retrieve all the streamer data
+        # !!! NOTE: streamer seems to retrieve a maximum of 100 items at a time
+        #           so we need to check that we have all the data
+        LOGGER.info(len(item.data))
+        feed_data.extend(item.data)
+        if feed_data.__len__() == streamer_list['Quote'].__len__():
+            break  # Stops the async for item in streamer.listen() loop after receiving all the data
+
+    # Process the streamer data to create the quotes list (not needed to get the greeks, just for example)
+    quotes = []
+    for data in feed_data:
+        # LOGGER.info('Symbol: {}\tBid: {}\tAsk {}'.format(data['eventSymbol'], data['bidPrice'], data['askPrice']))
+        # Create quote object
+        qd = Quote(data)
+        quotes.append(qd)
+
+    # Get the Greeks for all the options
+    # sub_greeks = {"Greeks": [".SPY210419P410"]}
+    streamer_list = {"Greeks": options_symbols}
+    await streamer.add_data_sub(streamer_list)
+
+    greeks_data = []
+    async for item in streamer.listen():
+        # This is where you manipulate streamer data
+        LOGGER.info(len(item.data))
+        greeks_data.extend(item.data)
+        if greeks_data.__len__() == streamer_list['Greeks'].__len__():
+            break  # Stops the async for item in streamer.listen() loop after receiving all the data
+
+    for data in greeks_data:
+        gd = Greeks()
+        gd.from_dict(data)
+        idx_match = [x for x in range(options.__len__()) if options[x].symbol == gd.symbol][0]
+        options[idx_match].greeks = gd
+        LOGGER.info('> Symbol: {}\tPrice: {}\tDelta {}'.format(gd.symbol, gd.price, gd.delta))
 
+    ##########
+    # ORDERS #
+    ##########
+    # Execute an order
     details = OrderDetails(
         type=OrderType.LIMIT,
         price=Decimal(400),
         price_effect=OrderPriceEffect.CREDIT)
     new_order = Order(details)
 
     opt = Option(
-        ticker='AKS',
+        ticker='GME',
         quantity=1,
-        expiry=get_third_friday(date.today()),
-        strike=Decimal(3),
-        option_type=OptionType.CALL,
+        expiration_date=get_third_friday(date.today()),
```

Comment:
👍 , I think `expiration_date` is a better name than `expiry` here

Reply:
I'll submit a separate PR that changes all 'expiry' in all the files to 'expiration_date'